packages/desktop-client/src/components/modals/ConfirmTransactionEdit.js

```diff
@@ -0,0 +1,76 @@
+import Block from '../common/Block';
+import Button from '../common/Button';
+import Modal from '../common/Modal';
+import View from '../common/View';
+
+export default function ConfirmTransactionEdit({
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could you please convert it to typescript? We are slowly moving everything over to TS. All new components should be automatically created with TS.